class-cr-clone.php

```diff
@@ -64,7 +64,17 @@ public function __action_admin_post() {
 			wp_die( esc_html__( 'There was an error copying this post', 'clone-replace' ) );
 		}
 
-		wp_redirect( admin_url( "post.php?post={$post_id}&action=edit" ) );
+		// Modify individual query args for redirect URL.
+		$admin_redirect_url = add_query_arg(
+			apply_filters( 'CR_Clone_redirect_query_args', [
+				'post'   => $post_id,
+				'action' => 'edit',
+			] ),
+			admin_url( "post.php" )
+		);
+
+		// Change redirect url completely using the filter below.
+		wp_redirect( esc_url( apply_filters( 'CR_Clone_redirect_url', $admin_redirect_url ) ) );
```

Comment:
`esc_url()` should only be used when _outputting_ a URL in HTML, e.g. an `href` or `src` attribute in an HTML tag, as it encodes entities. `esc_url_raw()` should be used to escape a URL in every other circumstance (redirects, outputting in JavaScript, REST API responses, etc.)